components/prism-turtle.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.turtle = {
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'multiline-string': {
+		pattern: /"""(?:[^"]"?"?)*"""|'''(?:[^']'?'?)*'''/,
+		greedy: true,
+		alias: 'string',
+		inside: {
+			'comment' : {
+				pattern: /#.*/
+			}
+		}
+	},
+	'string': {
+		pattern: /"(?:[^\\"\r\n]|\\.)*"|'(?:[^\\'\r\n]|\\.)*'/,
+		greedy: true
+	},
+	'url': {
+		pattern: /<[^\s<>]*>/,
+		greedy: true
+	},
+	'function': /\b[^\s:]*:[^\s:.;,(){}[\]^]*/,
+	'number': /[+-]?\b\d+\.?\d*(?:e[+-]?\d+)?/i,
+	'punctuation': /[{}.,;()[\]]|\^\^/,
+	'boolean': /\b(?:true|false)\b/,
+	'keyword': /(?:\b(?:a|graph|base|prefix)|@prefix|@base)\b|=/i,
```

Comment:
According to [the grammar](https://www.w3.org/TR/turtle/#h3_sec-grammar-grammar), only `BASE` and `PREFIX` are case-insensitive.

Reply:
I split the keywords in regular and case-insensitive. Includes `GRAPH` from TriG.